xdp.cpp.user.props

```diff
@@ -5,6 +5,9 @@
     <ClCompile>
       <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
     </ClCompile>
+    <Link>
+      <AdditionalOptions>/NODEFAULTLIB:libucrtd.lib /DEFAULTLIB:ucrtd.lib</AdditionalOptions>
```

Comment:
FYI, these kind of changes are good examples of why you should have arm64 builds onboarded. I'm not 100% sure both of these work on arm64...